extensions/positron-python/.eslintrc

```diff
@@ -1,115 +0,0 @@
-        "no-bitwise": "off",
-        "no-dupe-class-members": "off",
-        "@typescript-eslint/no-dupe-class-members": "error",
-        "no-empty-function": "off",
-        "@typescript-eslint/no-empty-function": ["error"],
-        "@typescript-eslint/no-empty-interface": "off",
-        "@typescript-eslint/no-explicit-any": "error",
-        "@typescript-eslint/no-non-null-assertion": "off",
-        "no-unused-vars": "off",
-        "@typescript-eslint/no-unused-vars": [
-            "error",
-            {
-                "args": "after-used",
-                "argsIgnorePattern": "^_"
-            }
-        ],
-        "no-use-before-define": "off",
-        "@typescript-eslint/no-use-before-define": [
-            "error",
-            {
-                "functions": false
-            }
-        ],
-        "no-useless-constructor": "off",
-        "@typescript-eslint/no-useless-constructor": "error",
-        "@typescript-eslint/no-var-requires": "off",
-
-        // Other rules
-        "class-methods-use-this": ["error", { "exceptMethods": ["dispose"] }],
-        "func-names": "off",
-        "import/extensions": "off",
-        "import/namespace": "off",
-        "import/no-extraneous-dependencies": "off",
-        "import/no-unresolved": [
-            "error",
-            {
-                "ignore": ["monaco-editor", "vscode"]
-            }
-        ],
-        "import/prefer-default-export": "off",
-        "linebreak-style": "off",
-        "no-await-in-loop": "off",
-        "no-console": "off",
-        "no-control-regex": "off",
-        "no-extend-native": "off",
-        "no-multi-str": "off",
-        "no-shadow": "off",
-        "no-param-reassign": "off",
-        "no-prototype-builtins": "off",
-        "no-restricted-syntax": [
-            "error",
-            {
-                "selector": "ForInStatement",
-                "message": "for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array."
-            },
-            {
-                "selector": "LabeledStatement",
-                "message": "Labels are a form of GOTO; using them makes code confusing and hard to maintain and understand."
-            },
-            {
-                "selector": "WithStatement",
-                "message": "`with` is disallowed in strict mode because it makes code impossible to predict and optimize."
-            }
-        ],
-        "no-template-curly-in-string": "off",
-        "no-underscore-dangle": "off",
-        "no-useless-escape": "off",
-        "no-void": [
-            "error",
-            {
-                "allowAsStatement": true
-            }
-        ],
-        "operator-assignment": "off",
-        "strict": "off",
-        "no-only-tests/no-only-tests": ["error", { "block": ["test", "suite"], "focus": ["only"] }]
-    },
-    // --- Start Positron ---
```

Comment:
Do we need this in `eslint.config.mjs`?

Reply:
Oh yeah, I made a note to look at this but forgot. It seems that this is no longer a problem since linting is passing.

Actually it looks like they turned off the no-explicit-any rule for all `.ts` files! It used to be
https://github.com/posit-dev/positron/blob/72b8d5bc4c5b1e2c68a4867071a47f57b96680eb/extensions/positron-python/.eslintrc#L33
but now it's
https://github.com/posit-dev/positron/blob/f01a7f422f3d91994d12fe3d412282ba387e8653/extensions/positron-python/eslint.config.mjs#L302